integration_tests/dbt_project.yml

```diff
@@ -34,9 +34,9 @@ sources:
       spark_external:
         +enabled: "{{ target.type == 'spark' }}"
       synapse_external:
-        +enabled: "{{ target.name == 'synapse' }}"
+        +enabled: "{{ target.type == 'synapse' }}"
       azuresql_external:
-        +enabled: "{{ target.name == 'azuresql' }}"
+        +enabled: "{{ target.type == 'azuresql' }}"
```

Comment:
I think this will want to be `target.name == 'azuresql'` or `target.type == 'sqlserver'`